js/sponsor.js

```diff
@@ -0,0 +1,26 @@
+$(document).ready(()=>{
```

Comment:
Again don't use arrow functions since it is introduced in ES6 and is not supported by all browsers yet. We are not doing any compilation. Also use vars for variable declarations.

Reply:
Cool..Thanks for patiently handling this PR